src/main/java/entities/computers/ModelOneLaptop.java

```diff
@@ -0,0 +1,58 @@
+package entities.computers;
+
+import interfaces.Computer;
+import interfaces.Monitor;
+import interfaces.Processor;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.extern.log4j.Log4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.beans.factory.config.ConfigurableBeanFactory;
+import org.springframework.context.annotation.PropertySource;
+import org.springframework.context.annotation.Scope;
+import org.springframework.stereotype.Component;
+
+import javax.annotation.PostConstruct;
+import javax.annotation.PreDestroy;
+
+@Getter
+@Setter
+@Log4j
+@Component
+@Scope(value = ConfigurableBeanFactory.SCOPE_SINGLETON)
```

Comment:
Scope singleton is the default value. So in this case annotation @Scope is redundant here.